scripts/run_tests.sh

```diff
@@ -68,10 +68,13 @@ docker run --rm -d -l ${LABEL} --name $RANDOM_NAME -p $PORT:5432 -e POSTGRES_USE
 # Set the DATABASE_URL environment variable
 export DATABASE_URL="postgres://postgres:postgres@localhost:${PORT}/scheduler"
 
+# Sleep
+sleep 1
+
 # Run the migrations
 cd crates/infra && sqlx migrate run && cd ../..
 
 # Run the tests
-cargo nextest run --workspace && cd ..
+cargo test --workspace && cd ..
```

Comment:
Unfortunately the tests should be run sequentially per crate (nextest run all crates in parallel), so for the moment I revert back to `cargo test` (default test runner)